### ISSUE TITLE
Classes gufo:SemiRigidType and gufo:Mixin

### ISSUE THREAD

**Issue by pedropaulofb** [nemo-ufes/gufo#6]
**Describe the bug**

The current version of gUFO (1.0.0) has the following statement: `gufo:Mixin rdfs:subClassOf gufo:NonSortal`
However, as there is no other gufo:NonSortal subclass, aren't these classes the same? In this case, they should be related through an owl:sameAs statement?

**Expected behavior**
Old statement: `gufo:Mixin rdfs:subClassOf gufo:NonSortal`
New statement: `gufo:Mixin owl:sameAs gufo:NonSortal`

**Comment by jpalmeida** [comment]
No, a gufo:Mixin is a gufo:EndurantType that is both non-sortal and semi-rigid. There are other subclasses of gufo:NonSortal  (e.g., gufo:Category).

**Comment by pedropaulofb** [comment]
@jpalmeida , I'm sorry! I wrote the correct class name in this issue title, but I wrote it wrong in the description. I was referring to the class `gufo:SemiRigidType`, not the class `gufo:NonSortal`.

I understand why there is a `gufo:SemiRigidType` and that this class does not have to do with sortality, but with rigidity. In this case, maybe it would be a possibility to just remove it and use Mixin directly as a subclass of gufo:NonRigidType.

To be honest, I am not sure if the modification here proposed would be good or not: it would turn the rigidity structure less clear, but maybe it would be a more correct representation.

**Now:**
```
gufo:EndurantType
	gufo:NonRigidType
		gufo:AntiRigidType
		gufo:SemiRigidType
			gufo:Mixin
	gufo:NonSortal
	gufo:RigidType
	gufo:Sortal
```

**Suggestion:**
```
gufo:EndurantType
	gufo:NonRigidType
		gufo:AntiRigidType
		gufo:Mixin
	gufo:NonSortal
	gufo:RigidType
	gufo:Sortal
```
